lib/telemetry.go

```diff
@@ -225,7 +225,7 @@ func (c *TelemetryConfig) MergeDefaults(defaults *TelemetryConfig) {
 				continue
 			}
 		case reflect.Bool:
-			if f.Bool() != false {
+			if f.Bool() {
```

Comment:
This line made me laugh a little. Good catch.

Reply:
Hah, yeah. I've seen `foo == true` a bunch of time, but `foo != false` is new :laughing: